ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/vlan-interfaces.yml

```diff
@@ -195,6 +195,17 @@ vlan_interfaces:
         preferred_lifetime: infinite
         no_autoconfig_flag: true
     ipv6_virtual_router_address: 1b11:3a00:22b0:1000::1
+    multicast:
+      ipv4:
+        static: true
+        boundaries:
+          - name: 224.0.1.0/24
+            out: true
+          - name: 224.0.2.0/24
+      ipv6:
+        boundaries:
```

Comment:
why do we use a different model for ipv6 vs ipv4? On my test DUT the CLI is the same.

Reply:
I see that `out` is always true for ipv6 in current eos, but I think we should keep the name style with dict and just skip the `out` key.